Libraries/LibCore/ProcessStatisticsReader.h

```diff
@@ -85,10 +85,11 @@ struct ProcessStatistics {
 class ProcessStatisticsReader {
 public:
     static HashMap<pid_t, Core::ProcessStatistics> get_all();
+    static Optional<String> get_current_working_directory(pid_t pid);
+    static Optional<String> get_executable_path(pid_t pid);
```

Comment:
`String` already has a null state and we use that instead of wrapping them in `Optional`.